.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
maybe you can skip  the contents of `.idea/*` folder

Reply:
Yeah we need to add it gitignore as others may also use inte